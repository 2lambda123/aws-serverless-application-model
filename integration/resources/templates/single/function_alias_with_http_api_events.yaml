Resources:
  HttpApi:
    Type: AWS::Serverless::HttpApi
  MyLambdaFunction:
    Type: AWS::Serverless::Function
    Properties:
      AutoPublishAlias: live
      Handler: index.handler
      Runtime: python3.8
      CodeUri: ${codeuri}
      Events:
        FooEvent:
          Type: HttpApi
          Properties:
            ApiId:
              Ref: HttpApi
        BarEvent:
          Type: HttpApi
          Properties:
            ApiId:
              Ref: HttpApi
            Path: /bar
            Method: ANY