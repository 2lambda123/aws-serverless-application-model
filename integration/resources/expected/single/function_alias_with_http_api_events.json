[
  { "LogicalResourceId":"MyLambdaFunction", "ResourceType":"AWS::Lambda::Function" },
  { "LogicalResourceId":"MyLambdaFunctionAliaslive", "ResourceType":"AWS::Lambda::Alias" },
  { "LogicalResourceId":"MyLambdaFunctionRole", "ResourceType":"AWS::IAM::Role" },
  { "LogicalResourceId":"MyLambdaFunctionVersion", "ResourceType":"AWS::Lambda::Version" },
  { "LogicalResourceId":"MyLambdaFunctionFooEventPermission", "ResourceType":"AWS::Lambda::Permission" },
  { "LogicalResourceId":"MyLambdaFunctionBarEventPermission", "ResourceType":"AWS::Lambda::Permission" },
  { "LogicalResourceId":"HttpApi", "ResourceType":"AWS::ApiGatewayV2::Api" },
  { "LogicalResourceId":"HttpApiApiGatewayDefaultStage", "ResourceType":"AWS::ApiGatewayV2::Stage" }
]
